### PR TITLE
docs: Add missing `ui_font_size` option in `configuring-zed.md`

### PR DESCRIPTION
**Pull request by jsparkdev**
Added `ui_font_size`, an option that works in the editor but is missing from the documentation.

Release Notes:

- N/A

### CODE REVIEW

File: docs/src/configuring-zed.md [hunk 1]
```diff
@@ -2128,6 +2128,16 @@ Float values between `0.0` and `0.9`, where:
...
 
+## UI Font Size
+
+- Description: The default font size for text in the UI.
+- Setting: `ui_font_size`
+- Default: `16`
+
+**Options**
+
+`integer` values
```
Comment (SomeoneToIgnore) [inline docs/src/configuring-zed.md:2139]: This is not complete, as there are limits: https://github.com/zed-industries/zed/pull/17829
Let's mention these also?

**Comment by jsparkdev** [comment]
@SomeoneToIgnore Do you mean to replace the integer value with an integer value between 6 and 100?

**Comment by SomeoneToIgnore** [comment]
Yes? What else suits there based on https://github.com/zed-industries/zed/pull/17829?
Also, I'd mention the units of measure, `px`/`pixels` there.

**Review by SomeoneToIgnore — APPROVED** [review]
Neat!
Thank you so much for paying attention to all this.

**Pull request merged**